safe/reportv4/impact_report.py

```diff
@@ -281,6 +281,8 @@ def __init__(
         self._aggregation_impacted = (
             aggregation_impacted or
             self._impact_function.aggregation_impacted)
+        if extra_layers is None:
```

Comment:
The `None` here is only a way to make the default argument is a mutable object. We can change it with anything immutable.